packages/@uppy/dashboard/src/style.scss

```diff
@@ -707,7 +707,7 @@ a.uppy-Dashboard-poweredBy {
 
   .uppy-size--md & {
     width: 100%;
-    height: 100px;
+    height: 129px;
     border: 0;
```

Comment:
`129px` vs `130px`? I’m guessing there’s a reason, that’s why I’m asking, so I can keep that in the back of my head for future refactors.

Reply:
Exactly, 129px is intentional, in that it's exactly the height that makes 1-line filenames and 2-line filenames not affect the height of the image. Here is an exaggerated example of height>129px:
![image](https://user-images.githubusercontent.com/7578559/58265825-df387c00-7d99-11e9-8ce4-76c1c74aa613.png)